tests/augmentation/test_augmentation.py

```diff
@@ -3291,6 +3292,83 @@ def test_same_on_batch(self, device, dtype):
+    @pytest.mark.parametrize(
+        "expected",
+        [
+            torch.tensor(
+                [
+                    [
+                        [
+                            [0.726599991321564, 1.000000000000000, 0.726599991321564],
+                            [0.662100017070770, 0.912100017070770, 0.662100017070770],
+                            [0.500000000000000, 0.691100001335144, 0.500000000000000],
+                        ],
+                        [
+                            [0.726599991321564, 1.000000000000000, 0.726599991321564],
+                            [0.662100017070770, 0.912100017070770, 0.662100017070770],
+                            [0.500000000000000, 0.691100001335144, 0.500000000000000],
+                        ],
+                        [
+                            [0.726599991321564, 1.000000000000000, 0.726599991321564],
+                            [0.662100017070770, 0.912100017070770, 0.662100017070770],
+                            [0.500000000000000, 0.691100001335144, 0.500000000000000],
+                        ],
+                    ]
+                ]
+            )
+        ],
+    )
+    def test_smoke(self, expected, device, dtype):
+        torch.manual_seed(1)
+        input_tensor = torch.ones(1, 3, 3, 3, device=device, dtype=dtype) * 0.5
+        expected = expected.to(device, dtype=dtype)
+        aug = RandomGaussianIllumination(gain=0.5, p=1.0)
+        res = aug(input_tensor)
+        assert input_tensor.shape == res.shape
+        self.assert_close(res, expected, rtol=1e-4, atol=1e-4)
+
+    def test_exception(self, device, dtype):
+        with pytest.raises(ValueError, match="sign must be a tuple or a float"):
+            RandomGaussianIllumination(sign=3)
+
+        with pytest.raises(ValueError, match="center must be a tuple or a float"):
+            RandomGaussianIllumination(center=3)
+
+        with pytest.raises(ValueError, match="sigma must be a tuple or a float"):
+            RandomGaussianIllumination(sigma=[0.01, 0.06])
+
+        with pytest.raises(ValueError, match="gain must be a tuple or a float"):
+            RandomGaussianIllumination(gain=[0.01, 0.06])
+
+        with pytest.raises(Exception, match="gain values must be between 0 and 1. Recommended values less than 0.2."):
+            RandomGaussianIllumination(gain=(0.01, 2))
+
+        with pytest.raises(Exception, match="sigma of gaussian value must be between 0 and 1."):
+            RandomGaussianIllumination(sigma=(0.01, 2))
+
+        with pytest.raises(Exception, match="center of gaussian value must be between 0 and 1."):
+            RandomGaussianIllumination(center=(0.01, 2))
+
+        with pytest.raises(Exception, match="sign of gaussian value must be between -1 and 1."):
+            RandomGaussianIllumination(sign=(0.01, 2))
+
+    @pytest.mark.parametrize("batch_shape", [1, 3, 3, 5])
+    @pytest.mark.parametrize("channel_shape", [1, 1, 2, 3])
```

Comment:
```suggestion
    @pytest.mark.parametrize("batch_shape", [1, 3, 5])
    @pytest.mark.parametrize("channel_shape", [1, 2, 3])
```
using separated `parametrize` will create a product between the cases, instead of being aligned/sequential... if you want to control the value pairs, you will need something like

```python
    @pytest.mark.parametrize("channel_shape,batch_shape", [(1, 1), (3, 2), (5, 3)])
```